src/components/HamburgerButton.js

```diff
@@ -0,0 +1,56 @@
+import {useState} from "react";
```

Comment:
didnt we talk about editing your prettier config file to use spaces between brackets? 
`{ useState }` vs `{useState}`